hrms/hr/doctype/interview/interview.py

```diff
@@ -97,6 +97,17 @@ def reschedule_interview(self, scheduled_on, from_time, to_time):
 
 		recipients = get_recipients(self.name)
 
+		reminder_settings = frappe.db.get_value(
+			"HR Settings",
+			"HR Settings",
+			["send_interview_reminder"],
+			as_dict=True,
+		)
+
+		if not cint(reminder_settings.send_interview_reminder):
```

Comment:
```suggestion
		if not cint(frappe.db.get_single_value("HR Settings", "send_interview_reminder")):
			return
```

Simplify this
Also move the `msgprint` before this line, so that we don't have to write it again on line number 133